### PR TITLE
FHAC-1509 unused code

### PR DESCRIPTION
**Pull request by trantang**

### CODE REVIEW

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docregistry/adapter/AdapterComponentDocRegistryOrchImpl.java [hunk 1]
```diff
@@ -747,14 +746,12 @@ public void loadResponseMessage(oasis.names.tc.ebxml_regrep.xsd.query._3.AdhocQu
                 if (NullChecker.isNotNullish(doc.getPatientId())) {
                     // Commented call to apply HL7 Encoding to the patient id as Doc Repo now stores the patient id in
```
Comment (mhpnguyen) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/docregistry/adapter/AdapterComponentDocRegistryOrchImpl.java]: Remove this comment block.

File: Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/policy/SubjectHelper.java [hunk 2]
```diff
@@ -38,33 +38,26 @@
...
         subject.getAttribute().add(attrHelper.attributeFactory(UserHomeCommunityAttributeId, Constants.DataTypeString,
-                determineSendingHomeCommunityId(sendingHomeCommunity, assertion)));
+            determineSendingHomeCommunityId(sendingHomeCommunity, assertion)));
         return subject;
     }
 
     public SubjectType subjectFactoryReident(HomeCommunityType sendingHomeCommunity, AssertionType assertion) {
         SubjectType subject = new SubjectType();
         subject.setSubjectCategory(SubjectCategory);
         // removed as this causes the user-role-code to show up twice
```
Comment (mhpnguyen) [inline Product/Production/Common/CONNECTCoreLib/src/main/java/gov/hhs/fha/nhinc/transform/policy/SubjectHelper.java]: Remove this comment

File: Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/entity/proxy/EntityDocRetrieveProxyWebServiceSecuredImpl.java [hunk 3]
```diff
@@ -69,12 +69,7 @@ public RetrieveDocumentSetResponseType respondingGatewayCrossGatewayRetrieve(Ret
             message.setNhinTargetCommunities(targets);
             message.setRetrieveDocumentSetRequest(body);
 
-            /* TODO: WTH is this?
-             *
-             * SamlTokenCreator tokenCreator = new SamlTokenCreator();
-            Map requestContext = tokenCreator.CreateRequestContext(assertion, url, NhincConstants.DOC_QUERY_ACTION);
-
-            ((BindingProvider) port).getRequestContext().putAll(requestContext);*/
+            /* TODO: WTH is this? */
```
Comment (mhpnguyen) [inline Product/Production/Services/DocumentRetrieveCore/src/main/java/gov/hhs/fha/nhinc/docretrieve/entity/proxy/EntityDocRetrieveProxyWebServiceSecuredImpl.java]: remove this

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7DbParser201306.java [hunk 4]
```diff
@@ -121,7 +121,6 @@ public static PRPAIN201306UV02 buildMessageFromMpiPatients(List<Patient> patient
                 "PropertyAccessException - Default Assigning Authority property not defined in adapter.properties",
                 e);
             // CONNECT environment corrupt; return error response
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7DbParser201306.java]: Remove this

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7Parser201306.java [hunk 5]
```diff
@@ -123,7 +123,6 @@ public static PRPAIN201306UV02 buildMessageFromMpiPatient(Patients patients, PRP
                 "PropertyAccessException - Default Assigning Authority property not defined in adapter.properties",
                 e);
             // CONNECT environment corrupt; return error response
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/mpi/adapter/component/hl7parsers/HL7Parser201306.java]: remove this

File: Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/entity/OutboundPatientDiscoveryStrategy.java [hunk 6]
```diff
@@ -55,7 +55,6 @@ public void execute(Orchestratable message) {
             // shouldn't get here
             LOG.error("NhinPatientDiscoveryStrategy input Orchestratable was not an EntityPatientDiscoveryOrchestratable!!!");
             // throw new
```
Comment (mhpnguyen) [inline Product/Production/Services/PatientDiscoveryCore/src/main/java/gov/hhs/fha/nhinc/patientdiscovery/entity/OutboundPatientDiscoveryStrategy.java]: remove this

**Review by mhpnguyen — CHANGES_REQUESTED** [review]

**Review by mhpnguyen — APPROVED** [review]

**Review by ptambellini01 — APPROVED** [review]

**Pull request merged**